README.md

```diff
@@ -5,19 +5,20 @@ A node module to download Github assets for github releases. It will also uncomp
 ## Command line
 
 ```bash
-npm install -g @etclabscore/dl-github-release
+$ npm install -g @etclabscore/dl-github-release
 
-dl-github-release --help
+
+$ dl-github-release --help
 
 Usage: download-github-release [options] <user> <repo>
 
 Options:
   -V, --version                      output the version number
-  -o, --outputDir [output]           output directory [output] (default: "/Users/zb/Code/etclabs/dl-github-releases")
+  -o, --outputDir [output]           output directory [output] (default: \"/Users/zb/Code/etclabs/dl-github-releases\")
   -p, --includePre                   download prerelease
   -d, --includeDraft                 download draft releases
   -a, --filterAssetsByName <rexexp>  filter assets name
-  -z, --zipped                       don't extract zip files
+  -z, --zipped                       don\'t extract zip files
```

Comment:
![Screenshot_20190402_085826](https://user-images.githubusercontent.com/45600330/55408395-8b41b180-5525-11e9-8203-44c0741ea545.png)

escape chars come thru litrally, here and above in `outputDir` message.

Seems awkward for README. Should be TTY-only.

Reply:
I really am not understanding why it comes out so borked.

How do I fix this?

Reply:
okay i just removed syntax highlighting and it seems to look better now